bakerydemo/static/css/main.css

```diff
@@ -14,81 +14,182 @@
 /* 9. Miscellaneous/ Helper styles ------------------------------------------ */
 
 /* Global styles */
+/* CSS Variables */
+:root {
+  --black: #000000;
+  --dark: #333333;
+  --grey: #6e6e6e;
+  --border-grey: #E1DCD3;
+  --white: #ffffff;
+  --cream: #f5f3e9;
+  --light-brown: #87744f;
+  --mid-brown: #825600;
+  --dark-brown: #553801;
+  --orange: #c55302;
+  --dark-orange: #833701;
+  --font--primary: "Marcellus", sans-serif;
+  --font--secondary: "Open Sans", sans-serif;
+}
+
+/* Utilities */
```

Comment:
Utilities should be the very last thing in this file so they can override component styles. Probably doesn’t matter either way for `u-sr-only` but I think it’d be good for us to follow the pattern nonetheless.